### PR TITLE
Cosmetic updates to the route guide example

### PR DESCRIPTION
**Pull request by wonderfly**
The original PR:

https://github.com/grpc/grpc-go/pull/58

### CODE REVIEW

File: examples/route_guide/server/server.go [hunk 1]
```diff
@@ -162,8 +161,8 @@ func (s *routeGuideServer) loadFeatures(filePath string) {
 	if err != nil {
 		log.Fatalf("Failed to load default features: %v", err)
 	}
-	if err := json.Unmarshal(file, &(s.savedFeatures)); err != nil {
-		log.Fatalf("Failed to load default features: %v", err)
+	if err := json.Unmarshal(file, &s.savedFeatures); err != nil {
+		log.Fatal("Failed to load default features: %v", err)
```
Comment (wonderfly) [inline examples/route_guide/server/server.go]: This seems to be an incorrect diff. The real diff as shown in the commits is from 'Fatal' to 'Fatalf'.

File: examples/route_guide/client/client.go [hunk 2]
```diff
@@ -63,9 +63,9 @@ func printFeature(client pb.RouteGuideClient, point *pb.Point) {
 	feature, err := client.GetFeature(context.Background(), point)
 	if err != nil {
 		log.Fatalf("%v.GetFeatures(_) = _, %v: ", client, err)
-		return
+	} else {
```
Comment (iamqizhao) [inline examples/route_guide/client/client.go]: "else" is not needed.

**Comment by dsymonds** [comment]
LGTM (after @iamqizhao's comment is addressed).

**Comment by wonderfly** [comment]
Addressing @iamqizhao 's comment. PTAL.

**Pull request merged**